docs/developer/meta/config-specs.md

```diff
@@ -140,6 +140,19 @@ It also respects a few extra fields under the `value` attribute of each option:
 
 Use the `--sync` flag of the [config validation command](../ddev/cli.md#ddev-validate-config) to render the example configuration files.
 
+## Data model consumer
+
+The [model consumer][config-spec-model-consumer] uses each spec to render the [pydantic](https://github.com/samuelcolvin/pydantic) models
+that checks use to validate and interface with configuration. These are shipped with every Agent and individual Integration release.
```

Comment:
The "These" here is vague -- I can't tell if it means "the specs" or "the pydantic models" or "the checks"

Reply:
Fixed!